includes/apple-exporter/components/class-quote.php

```diff
@@ -264,6 +276,10 @@ private function _apply_hanging_punctuation( $text ) {
 		// Trim the fat before beginning.
 		$text = trim( $text );
 
+		// If using HTML format, also strip the beginning and ending paragraph tags.
+		$text = preg_replace( '/^<p>/i', '', $text );
+		$text = preg_replace( '/<\/p>$/i', '', $text );
```

Comment:
is it more performant to break out the removal of the beginning and ending <p> tags? just curious as opposed to matching `^<p>(.*?)<\/p>$` (i didn't test this regex) and replacing with `$1`

Reply:
Hm, I'm not really sure. I was thinking that since these comparisons were simple (involved no wildcards, fixed string position, replace with blank string) that this would be faster. I ran a simple comparison on regex101.com and matching just the first part is 6 steps, matching just the last part is 8 steps, for a total of 14 steps, vs. `/^<p>(.*)<\/p>$/gs`, which it's saying is 21 steps. Obviously, there's some overhead in running `preg_replace` more than once, so the complexity of the regular expression isn't the only factor here. I think I'll leave it as-is for simplicity.